assets/css/app.css

```diff
@@ -3,3 +3,4 @@
 @import "./phoenix.css";
 @import "./square.css";
 @import "./board.css";
+@import "./message.css";
```

Comment:
Do you need the `.css` extension with the import statements?

Reply:
I'm not sure. I can try without.

Reply:
I might be thinking of scss, but when using scss we don't need to

Reply:
I do need the extensions. :(